node/pkg/governor/governor.go

```diff
@@ -230,6 +305,19 @@ func (gov *ChainGovernor) initConfig() error {
 		}
 	}
 
+	for _, flowCancelConfigEntry := range flowCancelTokens {
+		addr, err := vaa.StringToAddress(flowCancelConfigEntry.addr)
+		if err != nil {
+			return err
+		}
+		key := tokenKey{chain: vaa.ChainID(flowCancelConfigEntry.chain), addr: addr}
+
+		// Only add flow cancelling for tokens that are already configured for rate-limiting.
+		if _, ok := gov.tokens[key]; ok {
+			gov.tokens[key].flowCancels = true
+		}
```

Comment:
Should we log here if the token isn't already being tracked? Rather than just moving on silently?

Maybe in both cases we can log to debug

Reply:
Yeah that's a good call out. It would be helpful to know if there's a configuration issue